src/libs/Geometry/geom_static.cpp

```diff
@@ -150,21 +162,20 @@ GEOM::GEOM(const char *fname, const char *lightname, GEOM_SERVICE &_srv, long fl
     }
     srv.free(rvb);
     // read vertices
-    auto *_colData = colData;
-    if (lightname != nullptr && flsz / 4 != nvertices)
+    if (colData.size() != nvertices)
     {
-        srv.free(colData);
-        lightname = nullptr;
+        colData.clear();
     }
+    auto _colData = colData.begin();
```

Comment:
Please, remove any underscore in the code you're modifying. this is a very bad style and should be avoided everywhere.
consider naming it colDataIt